hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/impl/OperationExecutorImpl.java

```diff
@@ -118,14 +121,29 @@ public OperationExecutorImpl(HazelcastProperties properties,
                                  NodeExtension nodeExtension,
                                  String hzName,
                                  ClassLoader configClassLoader) {
+        this(properties, loggerService, thisAddress, runnerFactory, nodeExtension, hzName, configClassLoader, null);
+    }
```

Comment:
I think we can get rid of this constructor, it's only used in one test anyway.

Reply:
Done